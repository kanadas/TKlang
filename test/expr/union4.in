(\x -> if x == (1 @ 5) then 1 @ true else if x == (2 @ 'a') then 2 @ 1 else 1 @ false) (1 @ 5)
