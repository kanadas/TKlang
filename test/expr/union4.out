expr/union4.in
1@True
